libs/universal/src/constants/universal-window.ts

```diff
@@ -58,6 +58,10 @@ const DB_REQUEST: IDBOpenDBRequest = {
 const SELF = ['frames', 'parent', 'self', 'top', 'window'];
 const WINDOW_HANDLER: ProxyHandler<Window> = {
     get: (windowRef, key: string) => {
+        if (typeof window !== 'undefined' && Reflect.has(window, key)) {
```

Comment:
Shouldn't it be `globalThis`? I believe it's already available in all our supported browsers